.gitignore

```diff
@@ -7,3 +7,7 @@
 /build
 /distr
 *.pem
+
+yarn.lock
```

Comment:
We need to decide if we either add yarn.lock to the repo to make sure everyone has the same version of dependencies OR use explicit versions in package.json for all dependencies (remove ^ or ~ from version numbers).

What do you think?

Reply:
Particular versions of packages would be better solution. We shouldn't trust external package developers.